tests/end_to_end/test_fim/test_fim_windows/data/playbooks/configuration.yaml

```diff
@@ -3,30 +3,6 @@
   hosts: agents
```

Comment:
```suggestion
  hosts: wazuh-windows
```

Reply:
Done in 32249dfa1d732e79983d294b7848c8f12e53a5cf